### ISSUE TITLE
Ability to select multiple environments at once

### ISSUE THREAD

**Issue by kyle-nahrgang** [a85/POSTMan-Chrome-Extension#7075]
**Is your feature request related to a problem? Please describe.**
I find myself needing multiple environments where several of the variables are the same (but not always the same). For example, I have an environment for some set of API parameters that also contains host parameters. Sometimes the host (local, qa, production, etc) will change but I want to keep the API parameters the same, or vice versa. 

**Describe the solution you'd like**
The ability to select multiple active environments at one time would effectively solve my problem of needing to keep multiple environments synced with each other. 

**Describe alternatives you've considered**
I was initially looking to contribute and implement this myself, but the application itself does not look to be open source. I had looked at various competing applications and none have this feature.

**Additional context**
Some extra credit would be to support environment templates so that a given request or collection could require that one of each environment type is selected.

**Comment by shamasis** [comment]
Something similar to this is on the cards for short to mid term roadmap. Will update here when things change.

**Comment by rayrad** [comment]
If it helps, I second this. We work with a bunch of APIs. Each endpoint requires various values specific to one of our clients. So we have to duplicate all of the client info across multiple environments so that we can test against localhost, staging, or production. You or someone else working on Postman has probably put a lot of thought into this already, but it seems to me this would require some form of environment grouping. Thus we could create one group for localhost/staging/prod, and another for client info. Each group would have a prefix code defined that is used to reference an environments within it. (i.e. `{{clients:somename}}`, or `{{env:localhost}}`). It would also be useful to scope environments to different levels of the collection/folder hierarchy, but that's probably outside the scope of this feature request.

**Comment by JoeScho** [comment]
Thirded! This would be super useful. I work with multiple APIs which are deployed in different regions and environments. So I end up with various env files such as dev_eu, dev_us, etc. 

It would be great to have a couple of selectors so I could select one for the environment, and the other for the region.

**Comment by thryyy** [comment]
Agreed, it would help me get rid of dummy requests I use for this purpose.

**Comment by alexandr-kotikhov** [comment]
+1

the same:
https://github.com/postmanlabs/postman-app-support/issues/3930
https://github.com/postmanlabs/postman-app-support/issues/1998
https://github.com/postmanlabs/postman-app-support/issues/7075
https://github.com/postmanlabs/postman-app-support/issues/4968
https://github.com/postmanlabs/postman-app-support/issues/2621

and related, which can be resolved by combining vars from several environments:
https://github.com/postmanlabs/postman-app-support/issues/4898
https://github.com/postmanlabs/postman-app-support/issues/10775
